vertical-pod-autoscaler/docs/features.md

```diff
@@ -50,4 +52,79 @@ To enable this feature, set the --round-cpu-millicores flag when running the VPA
 
 ```bash
 --round-cpu-millicores=50
-```
\ No newline at end of file
+```
+
+## In-Place Updates (`InPlaceOrRecreate`)
+
+> [!NOTE] 
```

Comment:
I'm not strongly opinionated on this suggestion, so do with it what you will:
```suggestion
> [!WARNING] 
```

It feels like it should be a warning and not a note to me, but your choice.